test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/CallCodeOperationSuite.java

```diff
@@ -51,7 +51,7 @@ HapiApiSpec verifiesExistence() {
                 .when()
                 .then(
                         contractCall(contract, "callCode", INVALID_ADDRESS)
-                                .hasKnownStatus(INVALID_SOLIDITY_ADDRESS),
+                                .hasKnownStatus(FAIL_INVALID),
```

Comment:
Calling a non existent contract w/o balance should either immediately return a true success (as though the code was `0x00` / evm `STOP), not a fail invalid.